tests/test_coordinates.py

```diff
@@ -11,6 +11,10 @@
     import spacepy.irbempy as ib
 except ImportError:
     pass #tests will fail, but won't bring down the entire suite
+try:
```

Comment:
I wonder about using a skip decorator on these, since astropy is optional. The reason for just forging ahead if irbem didn't import is because we still want to let other tests go ahead (to get that feedback in case there were other things wrong) but that's still a really pathalogical case, didn't seem worth skip-decorating the whole file. But for astropy coords, somebody might benefit from having a clean test run even without astropy as an optional dependency.

Reply:
Tests now decorated with `skipUnless`